src/h2_stream_set.erl

```diff
@@ -0,0 +1,855 @@
+-module(h2_stream_set).
```

Comment:
well make it http2_* for now and we can change them all later :stuck_out_tongue: